lib/web_ui/lib/src/engine/text_editing/text_editing.dart

```diff
@@ -674,14 +681,20 @@ class EditingState {
   /// valid selection range for input DOM elements.
   factory EditingState.fromFrameworkMessage(
       Map<String, dynamic> flutterEditingState) {
+    final String? text = flutterEditingState.tryString('text');
+
     final int selectionBase = flutterEditingState.readInt('selectionBase');
     final int selectionExtent = flutterEditingState.readInt('selectionExtent');
-    final String? text = flutterEditingState.tryString('text');
```

Comment:
Why was this moved?

Reply:
Talked offline; it's easier to see that selectionBase and selectionExtent are grouped and composingBase and composingExtent are grouped if text is isolated. No change in functionality, just readability